avaframe/com1DFA/com1DFA.py

```diff
@@ -770,6 +816,11 @@ def initializeParticles(cfg, releaseLine, dem, logName='', relThField=''):
         mPartArray = np.empty(0)
         aPartArray = np.empty(0)
         hPartArray = np.empty(0)
+        idFixed = np.empty(0)
+        if len(relThField) != 0 and cfg.getboolean('iniStep'):
+            # set release thickness to a constant value for initialisation
+            relRaster = np.where(relRaster > 0., cfg.getfloat('relTh'), 0.)
+            log.info('relTh!= 0, but relRaster set to relTh values here so that uniform number of particles created')
```

Comment:
debug level

Reply:
Thinking about this I would suggest to set this to warning - because if relThField provided - for the iniStep we set a constant value to initialize particles to get a uniform distribution - the correct mass is then set afterwards - hence particles with big diff in mass possible and to control the total amount of particles the relTh value in the ini file is then most important - hence a warning here might be useful as the relThField is not something we have in the ini and could mention this there